test/container/Dockerfile

```diff
@@ -11,7 +11,7 @@ FROM docker.io/library/golang:1.19 as golang
 
 FROM docker.io/library/registry:2.8 as registry
 
-FROM docker.io/bitnami/kubectl:1.24.8 as kubectl
+FROM docker.io/bitnami/kubectl@sha256:625467eb8c3a3d60232923404941c32e787eb9003e644d0fa8258b0efa7f6a7f as kubectl
```

Comment:
```suggestion
FROM docker.io/bitnami/kubectldocker.io/bitnami/kubectl:1.26@sha256:625467eb8c3a3d60232923404941c32e787eb9003e644d0fa8258b0efa7f6a7f as kubectl
```

Reply:
Cool!
I wasn't familiar with that format. Will take into account when implementing the changes related to issue #12622.
I've overidden the existing commit with your proposal.

Reply:
my apologies, my copy pasta was insufficient this morning.  I corrected the suggested changes.

Reply:
I noticed after responding =)